src/runtime_src/core/edge/skd/sk_daemon.cpp

```diff
@@ -19,13 +19,15 @@
 #include <dlfcn.h>
 #include <execinfo.h>
 #include <string.h>
+#include <cstdarg>
 #include <sys/wait.h>
 #include <sys/mman.h>
 #include <sys/prctl.h>
 
 #include "sk_types.h"
 #include "sk_daemon.h"
-#include "xclhal2_mpsoc.h"
+#include "core/common/config_reader.h"
```

Comment:
I happen to be touching sdk/CMakeLists.txt currently.   Adding and using symbols from this header files implies that you must explicitly link with xrt_coreutil.  Please add that link dependency to this pull request.

Reply:
done